msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -302,6 +305,185 @@ The JSON response body for Meeting Details API is as follows:
 } 
 ```
 
+## shareAppContentToStage API
+
+The `shareAppContentToStage` API enables you to share specific parts of your app to the meeting stage. The API is available through the Teams client SDK.
+
+### Prerequisite
+
+To use the `shareAppContentToStage` API, you must obtain the RSC permissions. In the app manifest, configure the `authorization` property, and the `name` and `type` in the `resourceSpecific` field. For example:
+
+```json
+"authorization": {
+    "permission": { 
+    "resourceSpecific": [
+      { 
+      "name": "MeetingStage.Write.Chat",
+      "type": "Delegated"
+      }
+    ]
+   }
+}
+ ```
+
+### Query parameter
+
+The `shareAppContentToStage` API includes the following parameters:
+
+|Value|Type|Required|Description|
+|---|---|----|---|
+|**callback**| String | Yes | Callback contains two parameters, error and result. The *error* can either contain an error of type *SdkError*, in case of an error, or null when share is successful. The *result* can either contain a true value, in case of a successful share, or null when the share fails.|
+|**appContentURL**| String | Yes | URL that will be shared on to the stage.|
+
+### Example
+
+The following codes provide examples of `shareAppContentToStage` API:
+
+# [C#](#tab/dotnet)
+
+Not available
+
+# [JavaScript](#tab/javascript)
+
+```typescript
+
+const appContentUrl = "https://www.bing.com/";
+
+microsoftTeams.meeting.shareAppContentToStage ((err, result) => {
+if(result) {
+  this.setState({ isAppSharing: true });
+ }
+if(err) {
+  this.setState({ sharingError: err, isAppSharing: false })
+ }
+}, appContentUrl); 
+
+```
+
+# [JSON](#tab/json)
+
+Not available
+
+---
+
+### Response codes
+
+The `shareAppContentToStage` API returns the following response codes:
+
+|Response code|Description|
+|---|---|
+| **500** | Internal error. |
+| **501** | API is not supported in the current context.|
+| **1000** | App does not have proper permissions to allow share to stage.|
+
+## getAppContentStageSharingState API
+
+The `getAppContentStageSharingState` API enables you to fetch information about apps' sharing on the meeting stage.
+
+### Query parameter
+
+The `getAppContentStageSharingState` API includes the following parameter:
+
+|Value|Type|Required|Description|
+|---|---|----|---|
+|**callback**| String | Yes | Callback contains two parameters, error and result. The *error* can either contain an error of type *SdkError*, in case of an error, or null when share is successful. The `appContentStageSharingState` can either contain an `AppContentStageSharingState` object, indicating successful retrieval, or null, indicating failed retrieval.|
```

Comment:
@kdugue-msft  Is the naming accurate here?
_The appContentStageSharingState can either contain an AppContentStageSharingState object, indicating successful retrieval, or null, indicating failed retrieval._

Reply:
Good catch, since we use `error` and `result`, we should be consistent. 

Suggestion: 
The *result* can either contain an AppContentStageSharingState object, indicating successful retrieval, or null, indicating failed retrieval.